File: cinema-backend/src/main/java/org/example/cinemabackend/user/application/dto/response/CinemaManagerResponse.java
package org.example.cinemabackend.user.application.dto.response;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.Builder;
import org.example.cinemabackend.user.core.domain.Role;

@Builder
public record CinemaManagerResponse(
        @NotBlank String firstName,
        @NotBlank String lastName,
        @NotBlank @Email String email,
        String managedCinemaName,
        Role role
) {
}

File: cinema-frontend/src/app/user/dtos/response/cinema-manager.response.ts
import {CinemaResponse} from "../../../cinema/dtos/response/cinema.response";

export class CinemaManagerResponse {
  constructor(
    readonly firstName: string,
    readonly lastName: string,
    readonly email: string,
    readonly managedCinema: CinemaResponse,
  ) {
  }
}
